### DISCUSSION TITLE
"Find References" Gone from Context Menu after updating eclipse version

### DISCUSSION THREAD

**Discussion by sdaimwood** [eclipse/lsp4e#1030] [Q&A]
A product I work on recently upgraded from Eclipse 4.23 (Java 11) -> 4.31 (Java 17), along with updating to the latest java 17 version for all of the plugins that we can, including the Generic Eclipse Editor and LSP4E which we use to implement language server-based editors. After the updates, our editors are no longer showing the "Find References" option in the context/right-click menu, but they do show a new option to "Open Call Hierarchy" which is disabled and not of particular use for the languages we're creating editors for anyway. 

Has anyone experienced an issue like this, or have any idea why it might be happening? I'll post the before and after of the context menus:

Before and After:

<img width="168" alt="image" src="https://github.com/eclipse/lsp4e/assets/93732205/fb3cfa95-4199-4a52-8a1e-b5bf5a5bbe16">
<img width="166" alt="image" src="https://github.com/eclipse/lsp4e/assets/93732205/01be969d-b488-4c88-a100-23729b66ed69">

**Comment by oleosterhagen** [comment]
With pull request https://github.com/eclipse-platform/eclipse.platform.ui/pull/1124 the visibility of the menu item _Find References_ has been reduced. But in the Generic Editor it should still be visible. You can verify this e.g. when opening a Typescript file.

I hope you have already resolved your issue. Otherwise, could you provide a minimal example to reproduce the problem?
